src/templates/finops-hub/main.bicep

```
@@ -29,6 +29,13 @@ param exportRetentionInDays int = 0
 @description('Optional. Number of months of cost data to retain in the ingestion container. Default: 13.')
 param ingestionRetentionInMonths int = 13
 
+@description('Optional. Remote storage account for ingestion dataset.')
```

Just to confirm, this is a remote hub for you to **PUSH** data to, right?
```suggestion
@description('Optional. Storage account to push data to for ingestion into a remote hub.')
```